demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -55,10 +55,12 @@ class ContentItem(BaseContent):
     is_test: bool = False
 
     @validator("path", always=True)
-    def validate_path(cls, v: Path) -> Path:
+    def validate_path(cls, v: Path, values) -> Path:
         if v.is_absolute():
             return v
-        return CONTENT_PATH / v
+        if not CONTENT_PATH.name:
+            return CONTENT_PATH / v
+        return CONTENT_PATH.with_name(values.get("source", "content")) / v
```

Comment:
is it `source` or `source_repo`?
also, when is the condition `not CONTENT_PATH.name` in the if block equal false?

Reply:
currently it's named `source`. Should we change to `source_repo`?

this blocks equals to False in our unit tests, where sometimes the content path is just `Path(".")`

Reply:
we have as source_repo, its the field i added, but it worked with source, and i tried with source_repo and it worked as well.

Reply:
My bad. This supposes to be `source_repo`. Changed.